crates/kernel/src/builtins/bf_num.rs

```diff
@@ -78,9 +78,10 @@ fn bf_random(bf_args: &mut BfCallState<'_>) -> Result<BfRet, BfErr> {
     }
 
     let mut rng = rand::thread_rng();
-    match bf_args.args[0].variant() {
-        Variant::Int(i) => Ok(Ret(v_int(rng.gen_range(0..*i)))),
-        Variant::Float(f) => Ok(Ret(v_float(rng.gen_range(0.0..*f)))),
+    match bf_args.args.first().map(|var| var.variant()) {
+        Some(Variant::Int(i)) if *i > 0 => Ok(Ret(v_int(rng.gen_range(1..=*i)))),
+        Some(Variant::Int(_)) => Err(BfErr::Code(E_INVARG)),
+        None => Ok(Ret(v_int(rng.gen_range(1..=2147483647)))),
```

Comment:
Do we like restricting random numbers to the max that Lambda provides? Or let it go up into all of `i64` instead?

Reply:
That's an interesting theological question. I _doubt_ there's code in people's code that makes assumptions about this, but who knows? It might be safest to leave it with this, and then if we need bigger ranges we can add a new argument to the builtin, or a new builtin.